store-service/src/main/java/com/stormeye/event/repository/BidsRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.stormeye.event.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import com.casper.sdk.model.common.Digest;
+import com.stormeye.event.service.storage.domain.Bids;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * The Spring Data JPA repository for {@link Bids} domain objects
+ */
+public interface BidsRepository extends PagingAndSortingRepository<Bids, Long>, JpaRepository<Bids, Long> {
+    Optional<List<Bids>> findByDeployHash(final Digest deployHash);
```

Comment:
Shouldn't we be using Pages?  And never user Optional on Lists. Spring Data will provide an empty list if not Bids are found